site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -1,11 +1,53 @@
 <div class="electronic_file">
     <b>Full path to the directory containing the autograding config.json file:</b><br>
-    See samples here: <a target=_blank href="https://github.com/Submitty/Tutorial/tree/master/examples">Submitty GitHub sample assignment configurations</a><br>
-    <kbd>/usr/local/submitty/more_autograding_examples/upload_only/config</kbd>  (an assignment without autograding)<br>
-    <kbd>/var/local/submitty/private_course_repositories/MY_COURSE_NAME/MY_HOMEWORK_NAME/</kbd> (for a custom autograded homework)<br>
-    <kbd>/var/local/submitty/courses/{{ semester }}/{{ course }}/config_upload/#</kbd> (for a web uploaded configuration)<br>
-
-    <input style='width: 83%' type='text' id='config_path' name='eg_config_path' value="{{ admin_gradeable.getEgConfigPath() }}" class="required" placeholder="(Required)" />
+    <fieldset>
+        <input type="radio" id="default_configs" class="eg_config" value="true" {% if which_option == 0 %} checked = "true" {% endif %}/> Default configs
```

Comment:
Give this class a better name.  Also, the `eg` prefix is unnecessary.  It was my mistake to add that prefix to everything.